csunplugged/topics/content/en/binary-numbers/unit-plan/unit-plan.yaml

```diff
@@ -26,8 +26,8 @@ how-binary-digits-work:
     - binary-correct-representation
     - no-physical-zeros-ones
   generated-resources:
-    sorting-network:
-      description: One per student.
+    binary-cards:
+      description: One set.
```

Comment:
Definitely should remain in the YAML file, as it's likely different lessons use an interactive differently.

For example:
- One per student.
- One per group.